w3-aggregations.md

```diff
@@ -0,0 +1,76 @@
+# UCAN Filecoin aggregations
+
+This document describes a simple UCAN service allowing an implementor to receive an aggregate of CAR files for inclusion in a Filecoin deal.
+
+## Capabilities
+
+The service exposes the following invokeable capabilities:
+
+### `aggregate/add`
+
+```json
+{
+  "iss": "did:web:web3.storage",
+  "aud": "did:web:spade.storage",
+  "att": [{
+    "with": "did:web:spade.storage",
+    "can": "aggregate/add",
+    "nb": {
+      "aggregate": { "/": "bagy...aggregate" }
+    }
+  }]
+}
+```
+
+Informs the service that an aggregate of CARs is ready for a Filecoin deal. Successful response implies that the aggregate has been accepted by the service to be added to a Filecoin deal.
+
+The `nb.aggregate` value is a CAR CID that refers to a "Ferry" aggregate, a collection of `dag-cbor` blocks with format:
+
+```json
+{
+  "link": { "/": "bag..." },
+  "size": 110101,
+  "md5": "md5...",
```

Comment:
we need to sync with riba on the need of this. It can potentially be used with client to guarantee data integrity. However, I assume spade will validate `commP`. So, data integrity will be checked there anyway.